tests/src/fixture.rs

```diff
@@ -336,6 +383,7 @@ pub async fn wait_for_sync(
     store: &SubgraphStore,
     hash: &DeploymentHash,
     stop_block: BlockPtr,
+    expect_healthy: bool,
```

Comment:
This `expect_healthy` param I added turned out not to be useful in this PR, since we don't actually fail the subgraph. And in a separate PR I've exposed errors here in a [different way](https://github.com/graphprotocol/graph-node/pull/4147/files#diff-d470d1203f4dd0118c3c5ea05685e7be336d1174aad8b4f8b49567cae94552eaR356-R361). So lets remove it.